src/plugins/intel_cpu/src/config.h

```diff
@@ -53,6 +53,9 @@ struct Config {
 
     void readProperties(const std::map<std::string, std::string> &config);
     void updateProperties();
+    // exports the properties which should be saved in model cache
```

Comment:
Export/Import API != model caching. So the statement is not valid: you are saving properties during model export, not caching.
Method name should be corrected as well.

Reply:
The method was removed because the way of saving parameters in the exported model was changed